inspirehep/modules/records/jsonschemas/records/hep.json

```diff
@@ -654,12 +641,6 @@
                         "description": "Just in case...",
                         "title": "Other IDs"
                     },
-                    "uuid": {
-                        "title": "UUID",
-                        "description": "Universally unique identifier of the author.",
-                        "type": "string",
-                        "pattern": "[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}"
-                    },
```

Comment:
Ok, then I'm readding a WIP tag and we can wait for @jmartinm to be back. If something else is blocked by this (I don't see how) I'll resurrect and integrate.